app/stv/stv.js

```diff
@@ -0,0 +1,370 @@
+          )} at iteration ${reverseIteration}`,
+        });
+
+        // As long as the reveseindex is still larger then 1 we can look further back
+        while (reverseIteration >= 1) {
+          // Find the log object for the last iteration
+          const logObject = log.find(
+            (entry) => entry.iteration === reverseIteration
+          );
+
+          // Find the lowest score (with regard to the alternatives in the actual iteration)
+          const iterationMinScore = Math.min(
+            ...minAlternatives.map((a) => logObject.counts[a.description] || 0)
+          );
+
+          // Find the candidates (in regard to the actual iteration) that has the lowest score
+          const iterationMinAlternatives = minAlternatives.filter(
+            (alternative) =>
+              (logObject.counts[alternative.description] || 0) <=
+              iterationMinScore + EPSILON
+          );
+
+          // If we are at iteration lvl 1 and there is still a tie we cannot do anything
+          if (reverseIteration === 1 && iterationMinAlternatives.length > 1) {
+            log.push({
+              action: 'TIE',
+              description:
+                'The backward checking went to iteration 1 without breaking the tie',
+            });
+            // Eliminate all candidates that are in the last iterationMinAlternatives
+            log.push({
+              action: 'MULTI_TIE_ELIMINATIONS',
+              alternatives: iterationMinAlternatives,
+              minScore: Number(minScore.toFixed(4)),
+            });
+            iterationMinAlternatives.forEach(
+              (alternative) => (doneAlternatives[alternative._id] = {})
+            );
+            break;
+          }
+
+          reverseIteration--;
+          // If there is a tie at this iteration as well we must continue the loop
+          if (iterationMinAlternatives.length > 1) continue;
+
+          // There is only one candidate with the lowest score
+          const minAlternative = iterationMinAlternatives[0];
+          if (minAlternative) {
+            log.push({
+              action: 'ELIMINATE',
+              alternative: minAlternative,
+              minScore: Number(iterationMinScore.toFixed(4)),
+            });
+            doneAlternatives[minAlternative._id] = {};
+          }
+          break;
+        }
+      } else {
+        // There is only one candidate with the lowest score
+        const minAlternative = minAlternatives[0];
+        if (minAlternative) {
+          log.push({
+            action: 'ELIMINATE',
+            alternative: minAlternative,
+            minScore: Number(minScore.toFixed(4)),
+          });
+          doneAlternatives[minAlternative._id] = {};
+        }
+      }
+      nextRoundVotes = votes;
+    }
+
+    // We filter out the alternatives of the doneAlternatives from the list of nextRoundVotes
+    votes = nextRoundVotes.map((vote) => {
+      vote['priorities'] = vote.priorities.filter(
```

Comment:
Consistency? (`['priorities'] === .priorities`)

Reply:
I agree that the use of `[' '`] vs `(.)` is a little inconsistent. Some of it is written by Jonas, some of it is written by me and some of it is written by the compiler 🙃 I can clean up that.